packages/math-input/src/components/tabbar/item.tsx

```diff
@@ -75,70 +76,146 @@ function imageTintColor(
     return color.offBlack64;
 }
 export type ItemState = "active" | "inactive" | "disabled";
-type Props = {
+type ArrowKeyTabItemProps = {
     onClick: () => void;
     itemState: ItemState;
     itemType: KeypadPageType;
+    focus: boolean;
+    setFocus: React.Dispatch<React.SetStateAction<number>>;
+    index: number;
 };
 
-class TabbarItem extends React.Component<Props> {
-    render(): React.ReactNode {
-        const {onClick, itemType, itemState} = this.props;
-        return (
-            <Clickable
-                onClick={onClick}
-                disabled={itemState === "disabled"}
-                aria-label={itemType}
-                style={styles.clickable}
-                aria-selected={itemState === "active"}
-                role="tab"
-            >
-                {({hovered, focused, pressed}) => {
-                    const tintColor = imageTintColor(
-                        itemState,
-                        hovered,
-                        focused,
-                        pressed,
-                    );
+type TabItemProps = {
+    onClick: () => void;
+    itemState: ItemState;
+    itemType: KeypadPageType;
+};
+
+export function ArrowKeyTabbarItem(
+    props: ArrowKeyTabItemProps,
+): React.ReactElement {
+    const {onClick, itemType, itemState, focus, setFocus, index} = props;
+
+    const ref = useRef(null);
 
-                    return (
+    useEffect(() => {
+        if (focus) {
+            // Move element into view when it is focused
+            // @ts-expect-error - TS2339 - Property 'current' does not exist on type 'never'.
+            ref.current?.focus();
```

Comment:
This is the TS error I was talking about. IIRC `@ts-expect-error` suppresses **all** TS errors. This was recently an issue that I ran into: someone did a find/replace and a very obvious bug made it to prod because someone had suppressed a totally unrelated error.